### PR TITLE
[Docs] Installation Guide updates - image update / flow polish / resources link updates

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Installation Guide updates 
- Updated Unity editor screenshots 
- Flow polish 
- Resources link updates (broken tutorials link, added MR Dev Days video links)

### CODE REVIEW

File: Documentation/Installation.md [hunk 1]
```diff
@@ -138,37 +138,31 @@ To create a **HoloLens application**, switch to the Universal Windows Platform:
...
 
-    ![TMP Essentials](../Documentation/Images/getting_started/MRTK_GettingStarted_TMPro.png)
+    <img src="../Documentation/Images/getting_started/MRTK_GettingStarted_TMPro.png" width="600">
 
     If you get such a prompt, select the "Import TMP essentials" button. "TMP Essentials" refers to Text Mesh Pro plugin, which some of the MRTK examples use for improved text rendering. (See [Text in Unity](https://docs.microsoft.com/windows/mixed-reality/text-in-unity) for more detailed information)
 
 1. Close the TMP dialog. After this you need to reload the scene. You can do this by double-clicking the scene in the Project tab.
 
-1. Uncheck or shrink the size of the 3d icons under the Gizmos tab in the Scene view to reduce scene clutter
+1. Uncheck(Unity 2019 or higher) or shrink the size of the 3d icons using the slider(Unity 2018) under the Gizmos tab in the Scene view to reduce scene clutter
```
Comment (keveleigh) [inline Documentation/Installation.md:165]:
```suggestion
1. Uncheck (Unity 2019 or higher) or shrink the size of the 3d icons using the slider (Unity 2018) under the Gizmos tab in the Scene view to reduce scene clutter
```

File: Documentation/Installation.md [hunk 2]
```diff
@@ -120,8 +120,8 @@ To create a **HoloLens application**, switch to the Universal Windows Platform:
     > * See the [profiles](Profiles/Profiles.md#hololens-2-profile) for more information on the differences between DefaultMixedRealityToolkitConfigurationProfile and DefaultHoloLens2ConfigurationProfile.
 
     You will then see the following in your Scene hierarchy:
-
-    ![MRTK Scene Setup](../Documentation/Images/MRTK_SceneSetup.png)
+
```
Comment (keveleigh) [inline Documentation/Installation.md:123]:
```suggestion

```

**Review by keveleigh — APPROVED** [review]

**Comment by cre8ivepark** [comment]
@keveleigh What's the best way to apply this update to the release branch as well?

**Comment by keveleigh** [comment]
>What's the best way to apply this update to the release branch as well?

@cre8ivepark I'll go ahead and rebase this PR now, so we can start in the release branch

**Pull request merged**